tests/remote/test_remote_decode.py

```diff
@@ -467,6 +457,31 @@ class RemoteAutoencoderKLHunyuanVideoTests(
     return_pt_slice = torch.tensor([0.1656, 0.2661, 0.3157, 0.0693, 0.1755, 0.2252, 0.0127, 0.1221, 0.1708])
 
 
+class RemoteAutoencoderKLWanTests(
+    RemoteAutoencoderKLVideoMixin,
+    unittest.TestCase,
+):
+    shape = (
+        1,
+        16,
+        3,
+        40,
+        64,
+    )
+    out_hw = (
+        320,
+        512,
+    )
+    endpoint = DECODE_ENDPOINT_WAN_2_1
+    dtype = torch.float16
```

Comment:
cc @yiyixuxu Currently the endpoint is running in float16 and output seems ok (on random latent), the examples use float32 but we noticed in the original code that everything is under bfloat16 autocast context. Can we check with the authors regarding the use of float32 for VAE?

Reply:
i haven't heard back from them, 
maybe let's generate some sample comparison and ask @asomoza to help decide if there is a quality difference